src/ConnectedMode.UnitTests/Migration/MigrationPromptTests.cs

```diff
@@ -125,11 +98,9 @@ public async Task MigrateButtonClicked_StartMigrationWizard()
 
             var migrationWizardController = new Mock<IMigrationWizardController>();
 
-            var serviceProvider = SetUpServiceProviderWithSolution();
```

Comment:
Most of the tests don't use the `ISolutionInfoProvider`, so I just removed it (previously, all of the tests needed a service provider that returned a VS service otherwise we'd get a null-ref exception).